### PR TITLE
{CI} Verify PR title, history notes to make it more standard to reduce manual effort during release.

### PR DESCRIPTION
**Pull request by wangzelin007**
**Description**

With this feature, we can automatically verify pull request titel and history notes.
image

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE:az command a: Make some customer-facing breaking change.
[Component Name 2] az command b:added some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI Test

**Comment by jiasli** [comment]
Please consider adding comments to `scripts/ci/check_pull_request.py` to make it more readable.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -23,13 +23,14 @@ jobs:
...
+        body=$(curl https://api.github.com/repos/$(Build.Repository.Name)/pulls/$(System.PullRequest.PullRequestNumber) | jq -r '.body')
+        echo $body
         if [ "$(System.PullRequest.TargetBranch)" != "release" ] && echo $title | grep -iqF hotfix:; then
           echo "Hotfix PR should target release branch."
           exit 1
         fi
-        [[ $title =~ ^(\[.*\]|\{.*\}).* ]] && exit 0
-        echo "Pull Request title should follow https://aka.ms/submitAzPR"
-        exit 1
+        python /home/vsts/work/1/s/scripts/ci/check_pull_request.py "$title" "$body"
```
Comment (jiasli) [inline azure-pipelines.yml:33]: We can use relative path, such as

https://github.com/Azure/azure-cli/blob/d85ce1fd1db6976a0ab0892e92980c2817cb8093/azure-pipelines.yml#L934

**Review by calvinhzy — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by necusjz — APPROVED** [review]

**Pull request merged**